chart/kubeapps/values.yaml

```diff
@@ -694,3 +694,8 @@ featureFlags:
   reposPerNamespace: true
   invalidateCache: true
   operators: false
+  additionalClusters: []
+  # additionalClusters:
+  # - name: test-cluster
+  #   apiServiceURL: https://172.18.0.3:6443
+  #   certificateAuthorityData: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUN5RENDQWJDZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwKY201bGRHVnpNQjRYRFRJd01EWXdNVEExTkRNMU1Wb1hEVE13TURVek1EQTFORE0xTVZvd0ZURVRNQkVHQTFVRQpBeE1LYTNWaVpYSnVaWFJsY3pDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTHhzCmxnNVFDS2p1VjIxMHBINFNRMHBaNkRSMytaU1lqaEdQaE1LRGVjZDVkTVZDK0hUV1RGcDNhR00yK01NM0JhaWIKNlNxcE9QaVNVdk5xc2RlOGtVOXRFd3JNVXMzZ1lsdkFNWk5yOHNPWVNXWm9FTE9lVGd3SlJhVkhIOHBxN3huYQowT2U3N3hsSGtvenB1K1NMcHo2UGxKYzdqc1k3L2ExTklyM2dNL0ZhVzlIQks1bW5ZeWl3UEt3SjBMNWFMY1RrCk5aV3VtVUhOZ2hET1BnaWZZRktOTEhONjVoTkFlbGY1dUFQcGVTOXh2SGZISE51OElYeUhJNEI0Qkx2RmdSQjUKM3ZQZmNFbll4UWUya21ueUFKd1hLTTBEeGdYd0Q1OTkzdDIyclJQQitrMzR5VkViWlJlS3VmQWs5Y29TMjFnbApFbVVmRlVpV0lPd0kxWnlFeEY4Q0F3RUFBYU1qTUNFd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCCi93UUZNQU1CQWY4d0RRWUpLb1pJaHZjTkFRRUxCUUFEZ2dFQkFBS2NYdEtYL0FqNUJKQnpOWHRLVVNDOFpmelUKa2FqNlVkbXpGUkJYMUFheG4xcXZrMlBWb3NBaXV2enpHcW5KR2JFRXFBZnFIYUJQaVp0dnhWMmZHMFZlV2J1agpra1NRcDBid0o3WDAwZjIyVmJHRGVENjRQU0VRbEk2Y3VSNFVad0x4ZFpwakppc0k4bVRIbk4zMXFEcnJ4SzYrCjdocm44b0VZbEtDNjVXaGg1ek84cXQ3SWZvZ2ZCN3pYQW5qaFFSdFEwcE9ZUGhLVHdUemd6TGwyS2tRWGxvZmEKdXlNQ2JjSFBvVkVCb1oyb0N3RTZxbHZzdkFIOWo2V2kvbFZOZlE3OWM3YjNSeVE0TUVsUXVaQklGckVidU9uVAp3MEhWTGh3NUVKSWxQdkI4TU9NaFhmY25Hb1ErUU5OYkhBSXhIWFRlNEdOaWExMG4yQm45NXJYYWlNVT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```

Comment:
I would use some different commented data:

```
name: second-cluster
apiServiceURL: https://second-cluster:6443
certificateAuthorityData: LS0tLS1CRUd....
```

(literally with the ellipsis, to not include the whole cert)

Should we also add a note first saying this doesn't work yet?

Also, you are missing a blank line at the end.

Reply:
Done and done :)